src/main/java/ca/corefacility/bioinformatics/irida/model/workflow/submission/AnalysisSubmission.java

```diff
@@ -69,6 +70,10 @@
 	@JoinTable(name = "analysis_submission_sequencing_object", joinColumns = @JoinColumn(name = "analysis_submission_id", nullable = false), inverseJoinColumns = @JoinColumn(name = "sequencing_object_id", nullable = false))
 	protected Set<SequencingObject> inputFiles;
 
+	@NotAudited
```

Comment:
Is there a particular reason why `automated` is `@NotAudited`?

Reply:
I had it that way because it couldn't be edited, but since it can be delete it should be audited.  Done in fa584b2